src/os/alert/alertmanagerinstance.js

```diff
@@ -0,0 +1,12 @@
+goog.module('os.alertManager');
+goog.module.declareLegacyNamespace();
+
+goog.require('os.alert.AlertManager');
+
+/**
+ * Global alert manager reference.
+ * @type {os.alert.AlertManager}
+ */
+const instance = os.alert.AlertManager.getInstance();
```

Comment:
Ideally I'd like to move this into `alertmanager.js` as:
```
const instance = new os.alert.AlertManager();
Object.freeze(instance);
```
Then add our own `os.alert.AlertManager#getInstance` function that returns the instance.

That would replace the `goog.addSingletonGetter` call, but would be a breaking change. My goal in this PR is to illustrate how we can translate files to modules without breaking changes.

Reply:
Scratch the `Object.freeze` line. That prevents modification of any properties on the class.